fuel-vm/src/interpreter/blockchain/smo_tests.rs

```diff
@@ -1,234 +0,0 @@
-    ; "address + call abi length overflows"
-)]
-#[test_case(
-    Input {
-        recipient_mem_address: 0,
-        msg_data_ptr: VM_MAX_RAM - 64,
-        msg_data_len: 100,
-        amount_coins_to_send: 0,
-        max_message_data_length: Word::MAX,
-        ..Default::default()
-    } => Err(RuntimeError::Recoverable(PanicReason::MemoryOverflow))
-    ; "address + call abi length overflows memory"
-)]
-#[test_case(
-    Input {
-        recipient_mem_address: 0,
-        msg_data_ptr: 0,
-        msg_data_len: 101,
-        amount_coins_to_send: 0,
-        max_message_data_length: 100,
-        ..Default::default()
-    } => Err(RuntimeError::Recoverable(PanicReason::MessageDataTooLong))
-    ; "call abi length > max message data length"
-)]
-#[test_case(
-    Input {
-        recipient_mem_address: 400,
-        msg_data_ptr: 0,
-        msg_data_len: 10,
-        amount_coins_to_send: 30,
-        initial_balance: 29,
-        ..Default::default()
-    } => Err(RuntimeError::Recoverable(PanicReason::NotEnoughBalance))
-    ; "amount coins to send > balance from external context"
-)]
-#[test_case(
-    Input {
-        recipient_mem_address: 400,
-        msg_data_ptr: 0,
-        msg_data_len: 10,
-        amount_coins_to_send: 30,
-        initial_balance: 29,
-        internal: true,
-        ..Default::default()
-    } => Err(RuntimeError::Recoverable(PanicReason::NotEnoughBalance))
-    ; "amount coins to send > balance from internal context"
-)]
-#[test_case(
-    Input {
-        recipient_mem_address: 400,
-        msg_data_ptr: 432,
-        msg_data_len: 10,
-        amount_coins_to_send: 20,
-        initial_balance: 29,
-        ..Default::default()
-    } => matches Ok(Output { external_balance: 9, internal_balance: 29, .. })
-    ; "coins sent succesfully from external context"
-)]
-#[test_case(
-    Input {
-        recipient_mem_address: 400,
-        msg_data_ptr: 432,
-        msg_data_len: 10,
-        amount_coins_to_send: 20,
-        initial_balance: 29,
-        internal: true,
-        ..Default::default()
-    } => matches Ok(Output { external_balance: 29, internal_balance: 9, .. })
-    ; "coins sent succesfully from internal context"
-)]
-#[test_case(
-    Input {
-        recipient_mem_address: 400,
-        msg_data_ptr: 432,
-        msg_data_len: 10,
-        amount_coins_to_send: 20,
-        initial_balance: 20,
-        ..Default::default()
-    } => matches Ok(Output { external_balance: 0, internal_balance: 20, .. })
-    ; "spend all coins succesfully from external context"
-)]
-#[test_case(
-    Input {
-        recipient_mem_address: 400,
-        msg_data_ptr: 432,
-        msg_data_len: 10,
-        amount_coins_to_send: 20,
-        initial_balance: 20,
-        internal: true,
-        ..Default::default()
-    } => matches Ok(Output { external_balance: 20, internal_balance: 0, .. })
-    ; "spend all coins succesfully from internal context"
-)]
-fn test_smo(
```

Comment:
why was this test removed? It has little to do with these memory changes and is more about verify how it affects runtime balances.